app/models/competition.rb

```diff
@@ -1155,15 +1141,15 @@ def pending_results_or_report(num_days)
   end
 
   # does the competition have this field (regardless of whether it's a date or blank)
-  def has_event_change_deadline_date?
+  def event_change_deadline_date_present?
```

Comment:
Maybe `..._required?`? I'm a bit dumb-struck myself because the previous version with `has_` was also not checking the field in question (`event_change_deadline_date`) at all...